src/game_events/pump.cpp

```diff
@@ -173,7 +173,7 @@ namespace { // Support functions
 	/**
 	 * Returns true iff the given event passes all its filters.
 	 */
-	bool t_pump::filter_event(const event_handler& handler, const queued_event& ev)
+	bool pump::filter_event(const event_handler& handler, const queued_event& ev)
```

Comment:
I don't think that "pump" is a good name, I'd rather see it be "game_event_pump" or something similar, otherwise it's unclear what kind of events that it's about.